charts/kubescape-sneeffer/values.yaml

```
@@ -0,0 +1,118 @@
+enabled: false
```

```suggestion
enabled: true
```
What is the point of running this chart if the default is false?